_includes/code/components/date-input.html

```diff
@@ -1,3 +1,3 @@
 {% capture date-input %}
-  {% fractal_component date-input %}
-{% endcapture %}{{ date-input | strip }}
\ No newline at end of file
+  {% library_component usa-memorable-date %}
```

Comment:
@mejiaj @thisisdano Noticed that this component is called two different things: `Date input` in our docs and `Memorable date` in our stories. Should we update docs to call this `Memorable date` to match the class name? Or call it `Date input` everywhere?

Reply:
Let's call it `Memorable date` throughout